src/modules/python/pbs/v1/_pmi_types.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, requested_pmi=None):
 
         try:
             _temp = __import__("pbs.v1._pmi_" + self.pmi_type,
-                               globals(), locals(), ['Pmi'], -1)
+                               globals(), locals(), ['Pmi'], 0)
```

Comment:
this is a change in behavior: In PY2 -1 in `__import__()` means both absolute and relative imports will be attempted. But PY3 0 means only perform absolute imports will be attempted.
references:
https://docs.python.org/2.7/library/functions.html#__import__
https://docs.python.org/3.6/library/functions.html#__import__
https://www.python.org/dev/peps/pep-0328

Reply:
According to the [document](https://docs.python.org/3/library/functions.html#__import__) -
For example, the statement import spam results in bytecode resembling the following code:
`spam = __import__('spam', globals(), locals(), [], 0)`
The statement import spam.ham results in this call:
`spam = __import__('spam.ham', globals(), locals(), [], 0)`
Since we are doing `import pbs.v1._pmi_`, we used 0.

Reply:
@npadole20 I understand what you are saying but here I am not talking about what we are importing and how we are importing but I am talking about from which directory to import. In PY2 when you pass -1 as the last argument if does absolute/relative import from all parent directory (till it finds module) but when you pass 0 it just looks into the current directory.

Reply:
I will create a separate ticket to handle this. Hope it works for you.